dvipdfmx.def

```diff
@@ -83,7 +83,7 @@
         background \current@color}}
 \def\define@color@named#1#2{%
   \expandafter\let\csname col@#1\endcsname\@nnil}
-      \def\no@page@color{\special{background \string"newpath clip}}
+\def\no@page@color{\special{background gray 1}}
```

Comment:
This is solid white I think but the intention here is to be "nothing" that is, a transparent background.

Reply:
No I checked the pdf: when adding a background xdvipdfmx adds an additional content object to page, but it suppress it when it is white. (I found that by chance)